CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 * Improved documentation about `dynamicType:` vs `type(of:)`.  
   [Antondomashnev](https://github.com/Antondomashnev)
   [#221](https://github.com/AliSoftware/OHHTTPStubs/pull/221)
+* Fixed a race condition that occasionally prevented redirect callbacks. Fixed response timing for zero-length stub data.  
```

Comment:
No, I appreciate this help! I haven't contributed to an open-source project in a long time and want to be sure that all of my changes fit the project perfectly.